samples/gui/README.md

```diff
@@ -76,7 +76,8 @@ docker build . -t <registry>/<namespace>/datauserclient:latest
 ## Deployment
   ```
 cd <root>>/samples/gui
-  
+
+export KUBE_NAMESPACE=<namespace>  
```

Comment:
Please add some documentation describing what this is.  I.e. The namespace in which the workload is running and point out that the GUI must run in that namespace.  Also, I would suggest perhaps calling the env variable WORKLOAD_NAMESPACE.

Reply:
Changed the name, added documentation.